docker/compose.yml

```diff
@@ -108,8 +108,7 @@ services:
       MAX_TXDATA_BYTES: 30000 #30 KBytes
       ZKGETH_LOWER_GAS_MARGIN_PERCENTS: 120
       ZKGETH_UPPER_GAS_MARGIN_PERCENTS: 300
-    entrypoint: ""
-    command: [ "sh", "/scripts/node.sh", "1337", "/genesis.json", "0x0", "0x1C9C380", "0xF4240" ]
```

Comment:
What is the benefit of removing command?

Reply:
Just 1 less key to define, no functional benefit

Reply:
iiuc, by removing the `command` and just use `entrypoint` could replace the default entrypoint `/bin/sh -c` with the node.sh command and hence avoid running the node.sh command under a parent shell process, @Filter94 correct?

Reply:
I'm not sure about that, I'd need to check